tests/resources/resolveimports/circularImport_1.cellml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<model xmlns="http://www.cellml.org/cellml/2.0#"
+	xmlns:cellml="http://www.cellml.org/cellml/2.0#"
+	xmlns:xlink="http://www.w3.org/1999/xlink" name="circularImport1">
+	<import xlink:href="circularImport_2.cellml">
+		<component name="i_am_cyclic" component_ref="c2" />
+	</import>
+	<component name="i_am_ok_but_my_sibling_is_cyclic"/>
+</model>
```

Comment:
I would replace tabs with 2 spaces.

Reply:
Same for `tests/resources/resolveimports/circularImport_3.cellml`.

Reply:
Otherwise, rather than having all the test resource CellML files under `tests/resources/resolveimports`, I would have them under `tests/resources/importer` since they relate to the `Importer` class (in the same way that we have resources under `tests/resources/generator` for our `Generator`-based tests).

Reply:
cbb69365